src/azure-cli/azure/cli/command_modules/vm/tests/latest/_test_util.py

```diff
@@ -0,0 +1,25 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from azure_devtools.scenario_tests import RecordingProcessor
+
+
+class TimeSpanProcessor(RecordingProcessor):
```

Comment:
Can your Process be utilized by others? If yes, can we move the Processor to https://github.com/Azure/azure-cli/blob/dev/src/azure-cli-testsdk/azure/cli/testsdk/utilities.py like [StorageAccountKeyReplacer](https://github.com/Azure/azure-cli/blob/dev/src/azure-cli-testsdk/azure/cli/testsdk/utilities.py#L62)?

Reply:
Currently, I don't want to put this one into azure-test-sdk. I don't know how general this format is. I don't think it's easy to share across all services. We can keep it here, if we use it in the future, we can consider move it again.